news/create-newsfile.py

```diff
@@ -41,6 +43,7 @@
 blocks = [
     ('Features', 'feature-*.md'),
     ('Bugfixes', 'bugfix-*.md'),
+    ('FilterX', 'filterx-*.md'),
```

Comment:
I agree that FilterX may require a separate paragraph in the NEWS file (at least until a lot of changes keep coming into it), but introducing it as a block next to "features, bugfixes, packaging, others" can be problematic, because those categories apply to FilterX too: we add features there, we fix bugs, etc.

Allowing the `fx-` prefix, like `fx-feature`, `fx-bugfix` might be a resolution for this concern, or we could just keep writing the entries as usual, but with the `filterx:` prefix in the title.

Or if we just want to separate new FilterX features, but not the bugfixes and other minor changes, the above could work as well.

Reply:
Good point, I agree.

I have added two sections: `FilterX features` and `FilterX bugfixes`. `FilterX other changes` might be another candidate for some entries, but we might be okay with putting those to `FilterX features`.